payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -152,19 +175,17 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         }
     }
 
-    @VisibleForTesting
-    fun fetchIsGooglePayReady() {
-        if (isGooglePayReady.value == null) {
-            if (args.isGooglePayEnabled) {
-                viewModelScope.launch {
-                    val isGooglePayReady = withContext(workContext) {
-                        googlePayRepository.isReady().first()
-                    }
-                    _isGooglePayReady.value = isGooglePayReady
-                }
-            } else {
-                _isGooglePayReady.value = false
-            }
+    fun setupGooglePay(activity: ComponentActivity) {
```

Comment:
I agree that that constructor would be better, but the point of this change is to use the public API since we're moving this code to a different module and can't use internal methods. Are you proposing we make that constructor public then?

Reply:
I won't change the constructor now, but will get back to this when going through the analytics events (MOBILESDK-345).